### DISCUSSION TITLE
[Feature] Select albums to share them

### DISCUSSION THREAD

**Discussion by pablo03v** [immich-app/immich#9725] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The only issue I found which relates to this is the bulk actions for images inside of an album. 

I would love to select multiple (500) albums so I can share them with another internal user :) 

If this is too heavy for the web-ui for now I'd be more than happy with a cli implementation.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by Thinkscape** [comment]
# Yes please!!! 

I've got ~100 albums, and want to share all of them with my family.
There is no reasonable way to do that right now, as it currently requires manually navigating to each and manually selecting each person to share with.

**Comment by SandraPietzsch** [comment]
I totaly agree. Since I use the Immich Cli to create and upload my Albums, it would be great if we can specify one or multiple User I want to share this album with, using an aditional Parameter.

**Comment by bo0tzz** [comment]
#8543
